hail/python/hailtop/batch/docs/service.rst

```diff
@@ -8,12 +8,12 @@ Batch Service
 What is the Batch Service?
 --------------------------
 
-Instead of executing jobs on your local computer (the default in Pipeline), you can execute
+Instead of executing jobs on your local computer (the default in Batch), you can execute
 your jobs on a multi-tenant compute cluster in Google Cloud that is managed by the Hail team
-and is called Batch. Batch consists of a scheduler that receives job submission requests
-from users and then executes jobs in Docker containers on Google Compute Engine VMs (workers)
-that are shared amongst all Batch users. A UI is available at `<batch.hail.is>`_ that allows a
-user to see job progress and access logs.
+and is called the Batch Service. The Batch Service consists of a scheduler that receives job
+submission requests from users and then executes jobs in Docker containers on Google Compute
+Engine VMs (workers) that are shared amongst all Batch users. A UI is available at `<batch.hail.is>`_
+that allows a user to see job progress and access logs.
```

Comment:
that url needs to be `https://batch.hail.is` or it will be interpreted as a relative link to a file named `batch.hail.is`